library/src/main/kotlin/com/vinted/actioncable/client/kotlin/ConnectionMonitor.kt

```diff
@@ -49,13 +44,12 @@ class ConnectionMonitor(
 
     fun start() {
         reset()
-        stoppedAt = 0L
```

Comment:
Maybe connectionTerminated = false needed here?

Reply:
Not needed since there is no way to re-establish connection after ```terminate()``` call. Also, ```false``` is default initial value of ```connectionTerminated```